### PR TITLE
add selectable activity sample

### LINKED ISSUE #1117: Sample: custom activity UI to select activity
Working in PR #1199

- [ ] Programmatically select an activity
- [x] Click to select an activity
- [x] Highlight after selection
- [x] Accessibility
- [ ] If activity can be selected programmatically, will be nice to scroll the activity into view on select

This will be done as part of the emulator customization story, not on main branch.

### PR DESCRIPTION
**Pull request by a-b-r-o-w-n**
closes #1117

TODO:
- [x] a11y

### CODE REVIEW

File: samples/16.customization-selectable-activity/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Selectable Chat Activity
+
+In this sample, we show how to add an activity middleware to allow the user to select an activity.
```
Comment (corinagum) [inline samples/16.customization-selectable-activity/README.md]: Can we add more detail in the `README.md`? Including `npm i`, what the user can try out when testing this sample, any steps you think are pertinent. :)

**Comment by corinagum** [comment]
Do we want to include a11y in this PR, as well as selecting an activity programmatically? If not, we shouldn't close #1117 or file a new bug. :)

File: samples/16.customization-selectable-activity/README.md (hunk 1 above)
Comment (a-b-r-o-w-n) [reply samples/16.customization-selectable-activity/README.md]: Will do!

**Comment by a-b-r-o-w-n** [comment]
> Do we want to include a11y in this PR, as well as selecting an activity programmatically? If not, we shouldn't close #1117 or file a new bug. :)

I'm not sure the programatic selection makes sense (there is another example showing integration with the redux store) which can be expanded to add the programatic selection. I am working on a11y now. I'll push up a new commit shortly.

**Comment by corinagum** [comment]
Created #1632 for discussion / tracking for a sample of programmatically selectable activities.

**Pull request merged**